scripts/start-aries-agentd.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
this script not needed we should create docker image include rest api binary

Reply:
similar thing we do in https://github.com/trustbloc/sidetree-mock/blob/master/images/sidetree-mock/Dockerfile

Reply:
Not sure if I can do it in same way, because we need to start rest server inside container, at max I can skip go build step here if I use docker file.

Reply:
Agreed, will fix it as part of #585